build.gradle

```diff
@@ -12,11 +12,11 @@
 
 buildscript {
     repositories {
-        jcenter()
+        maven { url  "http://palantir.bintray.com/releases" }
```

Comment:
You shouldn't need to add bintray here. Check out https://github.com/palantir/auth-tokens/pull/33/files#diff-c197962302397baf3a4cc36463dce5eaR14

You may need to add the gradle plugin repo